UnitTests/Tests/TRWeatherUpdateCacheSpec.m

```diff
@@ -29,6 +29,14 @@ - (NSURL *)latestWeatherUpdateURL;
     [[NSFileManager defaultManager] removeItemAtURL:weatherUpdateURLForTesting() error:nil];
 };
 
+NSDictionary* (^weatherConditionsWithTemperature) (NSNumber*) = ^NSDictionary* (NSNumber *temp) {
```

Comment:
> I would like not doing this.

And I'd prefer that we didn't merge code that doesn't conform to the style guide, even if it will be removed eventually.